src/librustc/hir/def_id.rs

```diff
@@ -1,5 +1,4 @@
 use crate::ty::{self, TyCtxt};
-use crate::hir::map::definitions::FIRST_FREE_DEF_INDEX;
```

Comment:
Is `FIRST_FREE_DEF_INDEX` used anywhere else?

Reply:
No - I'll remove it.